css/style.css

```diff
@@ -6,10 +12,39 @@ header {
   padding-bottom: 50px;
 }
 
-body {
-  font-family: "Poppins", "Helvetica Neue", sans-serif;
-  background-color: #f5f5f5;
+.mobile-nav {
+  display: none;
+  position: fixed; /* Stay in place */
+  top: 0; /* Stay at the top */
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  background-color: #6070ff; /* Black background with opacity */
+  opacity: 0.9; /* 90% opaque */
+}
+
+.mobile-nav-content {
+  color: #fff;
+  padding: 20px;
+  width: 95%;
   margin: 0;
+  text-align: left;
+  font-size: 40px;
+  font-weight: 400;
+}
```

Comment:
- [ ] Great job but I think that If I scroll i will see all the sections but depend on the Figma template I should just see the menu without scrolling in the side so kindly fix it to match the Figma.🔥🔥💪🏻💪🏻✅

Reply:
thanks @Reem-lab , solved this and i learnt something new while doing it. highly appreciated